### PR TITLE
Improve instructions a little bit

### PR DESCRIPTION
**Pull request by jlian**
## Purpose

*  Small readme improvement to clarify instruction and minimum version needed 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by jlian** [comment]
This fixes https://github.com/Azure-Samples/azure-serverless-iot-button/issues/5

File: README.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ If you do not have a button and don't want to use the community button simulator
 Now that you have an IoT Hub and a connected device, we need to write a simple Azure Function to process the data the devices are generating.
 
 1. Open Visual Studio 2017
-    * Make sure all updates have been installed for 2017 and the Azure Functions and WebJobs tools.  This sample relies on the latest templates
-1. **File** -> **New Project** and under **Visual C#** -> **Cloud** select **Azure Functions**
-1. Select the **Azure Functions v2 Preview** runtime in the dropdown and start with an **empty project**.  Leave the defaults as-is.
+1. **Help** -> **Check for updates**. Make sure all updates have been installed for 2017 and the Azure Functions and WebJobs tools.  This sample relies on the latest templates.
```
Comment (Karishma-Tiwari-MSFT) [inline README.md]: We really need these changes. Thanks. :)

File: README.md [hunk 2]
```diff
@@ -55,12 +55,12 @@ If you do not have a button and don't want to use the community button simulator
 Now that you have an IoT Hub and a connected device, we need to write a simple Azure Function to process the data the devices are generating.
 
 1. Open Visual Studio 2017
-1. **Help** -> **Check for updates**. Make sure all updates have been installed for 2017 and the Azure Functions and WebJobs tools.  This sample relies on the latest templates.
+1. **Tool** -> **Extensions and Updates** to [check your Azure Functions and Web Jobs Tools version](https://docs.microsoft.com/azure/azure-functions/functions-develop-vs#check-your-tools-version). Make sure you have at least **version 15.0.40502.0**. If not, [update your tools](https://docs.microsoft.com/azure/azure-functions/functions-develop-vs#update-your-tools).
```
Comment (Karishma-Tiwari-MSFT) [inline README.md]: 👍

**Comment by jeffhollan** [comment]
Thank You!

**Pull request merged**